src/app/clusters/access-control-server/access-control-server.cpp

```diff
@@ -127,196 +127,135 @@ struct AccessControlEntryCodec
         return CHIP_NO_ERROR;
     }
 
-    CHIP_ERROR Encode(TLV::TLVWriter & aWriter, TLV::Tag aTag) const
+    static CHIP_ERROR Convert(const AccessControl::Entry::Target & from, AccessControlCluster::Structs::Target::Type & to)
+    {
+        if (from.flags & AccessControl::Entry::Target::kCluster)
+        {
+            to.cluster.SetNonNull(from.cluster);
```

Comment:
This assumes that `to` was default-initialized, right?  As in, `to.cluster` is starting out null.  Should be documented.

Reply:
I'll remove that assumption.

Reply:
Added else clauses to set null otherwise.